src/components/entity/ha-chart-base.html

```diff
@@ -258,11 +265,25 @@
         hidden: chart.isDatasetVisible(i)
       })));
       this.set('unit', this.data.unit);
-    }
+    },
+    _formatTicks(ticks) {
+      return ticks.map((tick) => {
+        window.hassUtil.formatTime(tick.value);
+      });
+    },
     drawChart() {
       const data = this.data.data;
       const ctx = this.$.chartCanvas;
 
+      const timelineProto = Chart.scaleService.constructors.timeline.prototype;
+      if (timelineProto.convertTicksToLabels !== _formatTicks) {
+        timelineProto.convertTicksToLabels = _formatTicks;
```

Comment:
We should not do this here. We should do this inside `resources/ha-chart-scripts.html`.

Reply:
Should that be something like this?
```js
  (function() {
    const formatTicks = function(ticks) {
      return ticks.map((tick) => window.hassUtil.formatTime(tick.value));
    };

    const timelineProto = Chart.scaleService.constructors.timeline.prototype;
    timelineProto.convertTicksToLabels = formatTicks;

    const lineProto = Chart.scaleService.constructors.line.prototype;
    lineProto.convertTicksToLabels = formatTicks;
  })();
```

Reply:
We're using ES2015 so you can just use

```js
{
  const formatTicks = …
}
```

No need to assign temp variables for `timelineProto` since you only use it once